content/faq/resources-for-anvil-users.md

```diff
@@ -16,17 +16,28 @@ The current list of large-scale datasets which will be available through AnVIL a
 - eMERGE
 - CSER (coming soon)
 
-The AnVIL Data Dashboard provides a real time view of which data sets and individual projects are available on [AnVIL](/data).
+The AnVIL Data Dashboard provides a real time view of which data sets and individual projects are available
+on [AnVIL](https://anvilproject.org/data/consortia).
```

Comment:
Any content "refactored" is due to a change in the internal link from `/data` to `https://anvilproject.org/data/consortia` for AnVIL and `https://anvilproject.org/ncpi/data/studies` for NCPI - and also prettier updating the markdown. We will make a subsequent ticket to make the url host align with the environment.